src/structures/Guild.js

```diff
@@ -1195,6 +1198,15 @@ class Guild extends AnonymousGuild {
    *   .catch(console.error);
    */
   setChannelPositions(channelPositions) {
+    if (!deprecationEmittedForSetChannelPositions) {
+      process.emitWarning(
+        'This method (Guild#setChannelPositions) is deprecated. Use GuildChannelManager#setPositions instead.',
```

Comment:
Not gonna suggest this for all as that'd be too spammy but what do you think of this format?
```suggestion
        'This Guild#setChannelPositions method is deprecated. Use GuildChannelManager#setPositions instead.',
```

Reply:
> The Guild#setChannelPositions method is deprecated

Reply:
Fixed

Reply:
Sure, I can do that!